analysis/source/multivar_plot/multivar_plot.r

```diff
@@ -0,0 +1,19 @@
+setwd("~/GSlab/template/analysis")
```

Comment:
Setting the working directory can be useful when writing/debugging a script, but we don't want to do this in the final product. All of our scripts, and the paths stored in config_global, are configured to be run from and to be rooted in the `analysis` directory by default.